client/src/pages/FlightData/tabs/Quick.js

```diff
@@ -44,27 +44,27 @@ const Quick = () => {
 		>
 			<Column style={{ marginBottom: "1rem" }}>
 				<Row>
-					<Box label="Altitude" content={altitude} />
+					<Box label="Altitude" content={(Math.round(altitude * 100) / 100) + " ft"} />
 					<Row>
-						<Box label="Yaw" content={orientation.yaw} />
-						<Box label="Roll" content={orientation.roll} />
-						<Box label="Pitch" content={orientation.pitch} />
+						<Box label="Yaw" content={(orientation.yaw) + "\u00B0"} />
+						<Box label="Roll" content={(orientation.roll) + "\u00B0"} />
+						<Box label="Pitch" content={(orientation.pitch) + "\u00B0"} />
 					</Row>
 				</Row>
 				<Row>
-					<Box label="Ground Speed" content={groundSpeed} />
-					<Box label="Airspeed" content={airspeed} />
+					<Row>
+						<Box label="Latitude" content={(Math.round(latLong.lat * 100000000) / 100000000) + "\u00B0"} />
+						<Box label="Longitude" content={(Math.round(latLong.lon * 100000000) / 100000000) + "\u00B0"} />
+					</Row>
 				</Row>
 				<Row>
-					<Box label="Distance to Waypoint" content={distToWaypoint} />
-					<Box label="Battery" content={battery} />
+					<Box label="Throttle" content={(throttle) + ""} />
```

Comment:
I have no clue what the units for throttle are - I believe according to ardupilot, throttle is somewhere around 1200, with anything higher providing lift and anything lower tilting the plane down. The only reason I didn't add units was because I didn't know what units to use.